userspace/falco/falco.cpp

```diff
@@ -86,6 +86,7 @@ static void usage()
 	   " -h, --help                    Print this page\n"
 	   " -c                            Configuration file (default " FALCO_SOURCE_CONF_FILE ", " FALCO_INSTALL_CONF_FILE ")\n"
 	   " -A                            Monitor all events, including those with EF_DROP_SIMPLE_CONS flag.\n"
+	   " --alternate-lua-dir <path>    Specify an alternate path for loading falco lua files\n"
```

Comment:
```suggestion
	   " --alternate-lua-dir <path>    Specify an alternate path for loading Falco lua files\n"
```

Reply:
Added a suggestion to make Falco upper case. I know that there are other places where it's lower case but we are trying to slowly make everything adhere the branding guidelines.

https://github.com/falcosecurity/falco/tree/master/brand#yes


```
Writing about Falco
Yes

Notice the capitalization of the following terms.

    The Falco Project
    Falco

No

    falco
    the falco project
    the Falco project

```